### ISSUE TITLE
KA-52M has two radars. Please amend to one.

### ISSUE THREAD

**Issue by KLABDB3K** [PygmalionOfCyprus/cmo-db-requests#105]
### DB Selector

DB3K

### Affected DBID(s)

#5072 Ka-52M Hokum B - 2022, Modernized, Radar/EO Upgrade

### Summary of Changes

**Ka-52M entry has two radars but should only have the one AESA:**
Currently has:
V006 Rezets - (Ka-52M Nose Radar, L/Ka Band Slotted) Radar, Radar, FCR, Air-to-Air &
Air-to-Surface, Short-Range, Max range: 37 km
AND
V006 Rezets - (Ka-52M Nose Radar, X Band AESA) Radar, Radar, FCR, Air-to-Air & Air-to-Surface, Short-Range,
Max range: 37 km

In reality the full production version has just the one Rezet radar which is AESA.
The Rezets radar is an AESA not a conventional array and has better range than given in the DB3K. It's not short ranged at circa 80nm against a warship.
"It can detect a group of tanks from 45 km, a railway bridge from 100 km, and a destroyer-class warship from 150 km, according to the manufacturer. In the air-to-air mode, it can detect a fighter aircraft with a radar cross section of 3 sq m from up to 50 km head-on and a hovering helicopter from 20 km."

### Sources

https://www.janes.com/defence-news/news-detail/russias-ka-52m-combat-helicopter-to-receive-aesa-radar
https://defenceview.in/russia-offers-to-sell-ka-52m-attack-helicopter-who-are-potential-customers/

**Comment by PygmalionOfCyprus** [comment]
At some point there was talk of the Rezets system actually including a slotted mechanical radar, but that seems to have disappeared into the aether.

Anyway, I've removed the mechanical half from the Ka-52M and massaged values for the AESA piece to better simulate those ranges. Max range is now 80nm and range of acquisition of varyingly sized airborne/surface targets is as close as I can get it without knowing the actual sensor stats.

Fixed for 493.
